### PR TITLE
LOOP-957: Temporary sensor error

### PR DESCRIPTION
**Pull request by rickpasetto**
Still a bit unclear on the specs here, but this, I think, wires up the basic "temporary sensor" alert.
I'm having some trouble, still with figuring out the spec for the repeat and snooze frequency.  I'm hoping @nhamming can help :)

[LOOP-957](https://tidepool.atlassian.net/browse/LOOP-957)

### CODE REVIEW

File: DexcomCGMKit/DexcomAlertConfigurationManager.swift [hunk 1]
```diff
@@ -393,6 +393,8 @@ public class DexcomAlertConfigurationManager {
             if noReadingsAlertFirstDetectedAt == nil {
                 noReadingsAlertFirstDetectedAt = currentTime
             }
+        case .aberrationError(.sensorError):
+            scheduleNotification(forAlert: DexcomAlertConfiguration(type: .sensorTemporaryFailure, enabled: true, alwaysSoundEnabled: false, sound: .defaultSound))
```
Comment (nhamming) [inline DexcomCGMKit/DexcomAlertConfigurationManager.swift]: I think scheduling for 3 hours like described in the copy makes sense.

Comment (rickpasetto) [reply DexcomCGMKit/DexcomAlertConfigurationManager.swift]: I'm confused....the copy says that this error is a temporary issue, and the user should wait "up to 3 hours".  But...I didn't interpret that to mean that the _alert_ should repeat every 3 hours.  Sorry...what am I missing here?

**Comment by rickpasetto** [comment]
@nhamming just to be super-clear: I took our discussion in Slack (https://tidepoolteam.slack.com/archives/CG1MZQRJ5/p1582223999063900?thread_ts=1582140081.057100&cid=CG1MZQRJ5) as indication that you were okay with this implementation's approach, which is why I re-requested approval as-is.  If that's not the case, and I'm mistaken, let's try to chat live so I can understand what the situation is.

**Comment by rickpasetto** [comment]
@nhamming I checked how the G6 app behaves with the emulator.  As far as I can tell, the G6 app does not issue a User Notification for this error; it only shows up in the UI like this:
IMG_0296
Therefore, there is no "repeating" an alert... as far as I can tell, it just shows as a "Sensor Error" as long as the condition applies, and then goes away when the condition goes away.
Given all of that...is there any other work needed for this PR?

**Comment by nhamming** [comment]
@rickpasetto I think the only open item is whether or not `alwaysSoundEnabled` should be forced `false`. Currently notifications are only forced `true` (cannot be silenced) or respect the user setting. Given the G6 app doesn't use notifications for this case, I'm not sure how we decide this. Maybe input from product?

**Comment by rickpasetto** [comment]
Per [this Slack thread](https://tidepoolteam.slack.com/archives/CC3DA6CAX/p1582659713009100), apparently this implementation is not correct...there's still open questions about the design.  Not sure what the right process is...for now marking PR "invalid" (and possibly will close it).

**Pull request closed without merging**